requirements.txt

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
@muhammadadeeltajamul Was this file committed on purpose?

Reply:
@timmc-edx yes... This file was auto generated by pip-compile

Reply:
It was generated when I ran 

`make upgrade-package package=openedx-forum`

Reply:
Interesting, thanks! I'll look into it.